performance/tests/test_eqprs_api.py

```diff
@@ -0,0 +1,35 @@
+#from django.test import TestCase
+import json
+from django.urls import reverse
+from rest_framework.test import APITestCase,APIRequestFactory
+from rest_framework import status
+from performance.views import IndicatorReadOnlyModelViewSet
+from performance.serializer import IndicatorSerializer
+from performance.models import Indicator
+# Create your tests here.
+
+
+class indicator_API_Test(APITestCase):
+    list_url = reverse("indicator-list")
```

Comment:
what is this URL at the end of the day? I think it's worth hardcoding it here rather than using reverse() so that it isn't accidentally changed without noticing

Reply:
with my change suggestions in this PR , I think it will be something like `/performance/api/v1/eqprs`?